en/mkdocs.yml

```diff
@@ -419,7 +419,15 @@ nav:
           - developer-portal/applications/overview.md
           - 'Create New Application':
             - developer-portal/applications/new-application.md
-            - 'Single Page Application': developer-portal/applications/add/spa.md
+            - 'Single Page Application': 
+              - developer-portal/applications/add/spa.md
+              - 'SDKs': 
+                  - developer-portal/applications/sdks/overview.md
+                  - developer-portal/applications/sdks/java-android.md
+                  - developer-portal/applications/sdks/javascript-oidc.md
+                  - developer-portal/applications/sdks/react-auth.md
+                  - developer-portal/applications/sdks/windows.md
```

Comment:
Shall we add proper titles similar to the format of nav items after line 430? 

e.g.

- 'Android SDK': developer-portal/applications/sdks/java-android.md

Reply:
Resolved with b482634